### DISCUSSION TITLE
Format Documentation

### DISCUSSION THREAD

**Discussion by mblahay** [sql-formatter-org/sql-formatter#472] [Q&A]
Is there any formal documentation on what the expected format is for the output of the tool? Looking for something that can be referenced/reviewed in some project documentation.

**Comment by nene** [comment]
Well, the most format documentation we have is in the form of tests, but that's probably not what you want.

Other than that, there's the docs for [indentStyle](https://github.com/sql-formatter-org/sql-formatter/blob/master/docs/indentStyle.md) option.
